elastic_appsearch.theme.inc

```diff
@@ -0,0 +1,49 @@
+<?php
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\elastic_appsearch\Entity\Server;
+use Drupal\Core\Render\Element;
+use Drupal\Core\Url ;
+
+/**
+ * Prepares variables for elastic_appsearch_admin_fields_table form templates.
+ *
+ * Default template: elastic-appsearch-admin-fields-table.html.twig.
+ *
+ * @param array &$variables
+ *   Associative array of template variables, with the following structure:
+ *   - element: Associative array with the following keys:
+ *     - form: A render element representing the form.
+ *     - note: The table note.
+ */
+function template_preprocess_elastic_appsearch_admin_fields_table(array &$variables) {
+  $form = $variables['element'];
+  $rows = [];
+  if (!empty($form['fields'])) {
+    foreach (Element::children($form['fields']) as $name) {
+      $row = [];
+      foreach (Element::children($form['fields'][$name]) as $field) {
```

Comment:
Can this be changed from `foreach (Element::children($form['fields'][$name]) as $field) {` to `foreach ($name as $field) {` ?

Reply:
This is done to have the children fields appear on same row.

Reply:
ok